.github/workflows/flutter.yaml

```diff
@@ -0,0 +1,38 @@
+name: Dart
```

Comment:
We may need copyrights at the top of these files too.

Reply:
We do not have the in other workflows in dart repo: https://github.com/dart-lang/linter/blob/main/.github/workflows/build.yaml